salsa_webqa/library/control_test.py

```diff
@@ -47,20 +47,20 @@ def get_project_root(self):
         return project_root
 
     def load_configs(self):
-        """ Loads variables from .properties configuration files """
+        """ Loads variables from .properties configuration files,  check if project didn't contain such folder
+        (for non selenium projects) """
         config_path = os.path.join(self.project_root, 'config')
+        if not os.path.exists(config_path):
+            return None
```

Comment:
I'd personally put all the code below `return None` in the `else:` branch. I know it's not necessary in this, but I think it could be more readable that way. Just a suggestion though.